idaes/core/util/unit_costing.py

```diff
@@ -599,7 +652,7 @@ def CP_rule(self):
                             CE_index/500)*self.base_cost)
                 self.cp_cost_eq = Constraint(rule=CP_rule)
             else:
-                raise Exception('mover type not supported')
+                raise ValueError('mover type not supported')
```

Comment:
Suggest: `"{} - mover type '{}' not supported. Please see documentation for list of supported mover types.".format(self.name, mover_type)`